File: examples/golang/hellowithdeps.go
package kubeless

import (
	"github.com/kubeless/kubeless/pkg/functions"
	"github.com/sirupsen/logrus"
)

// Hello sample function with dependencies
func Hello(event functions.Event, context functions.Context) (string, error) {
	logrus.Info(event.Data)
	return "Hello world!", nil
}

File: examples/java/HelloWithDeps.java
package io.kubeless;

import io.kubeless.Event;
import io.kubeless.Context;

import org.joda.time.LocalTime;

public class Hello {
    public String sayHello(io.kubeless.Event event, io.kubeless.Context context) {
        System.out.println(event.Data);
        LocalTime currentTime = new LocalTime();
        return "Hello world! Current local time is: " + currentTime;
    }
}
